sympy/integrals/meijerint.py

```diff
@@ -1664,20 +1664,25 @@ def meijerint_indefinite(f, x):
     -cos(x)
     """
     f = sympify(f)
+    x_ = x
+    d = Dummy('x')
+    f = f.subs(x, d)
+    x = d
```

Comment:
```suggestion
    x_, x = x, Dummy('x')
    f = f.subs(_x, x)
```

Reply:
And add a comment here why we are doing this. Is the point to ignore assumptions?